### DISCUSSION TITLE
How to delete a blog page?

### DISCUSSION THREAD

**Discussion by BogusCurry** [datenstrom/community#224] [Ask a question] [answered]
Hi there ;D
I read the docu for the blog, but I can no read how I can delete a blogpost ? 

Greetings

Bogus

**Comment by BsNoSi** [comment, marked as answer]
I have to admit, that I never thought about that — I simply delete the file… .

I did not try but: after log in, you have a `+`and a `-`in the upper right corner. The `+`creates a new page, the `-` will delete (but I never used that).
